apps/posts/api/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path, include
+
+
+
+urlpatterns = [
+    
+]
```

Comment:
Is this a mistake?

Reply:
I think I've added stuff there in the second pr